### PR TITLE
Remove unnecessary int type conversion

### PR DESCRIPTION
**Pull request by sourcelliu**
#### What type of PR is this?
/kind cleanup
/sig node

#### What this PR does / why we need it:
Remove unnecessary int type conversion
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/kubelet/kuberuntime/kuberuntime_container.go [hunk 1]
```diff
@@ -136,7 +136,7 @@ func calcRestartCountByLogDir(path string) (int, error) {
 	if _, err := os.Stat(path); err != nil {
 		return 0, nil
 	}
-	restartCount := int(0)
+	restartCount := 0
```
Comment (bart0sh) [inline pkg/kubelet/kuberuntime/kuberuntime_container.go:139]: Would it make sense to move this line down, before the loop where this variable is used?

**Comment by bart0sh** [comment]
/triage accepted
/priority backlog
/assign

@sourcelliu please look for a similar code in the pkg/kubelet directory and fix it as well, thanks.

File: pkg/kubelet/kuberuntime/kuberuntime_container.go (hunk 1 above)
Comment (bart0sh) [reply pkg/kubelet/kuberuntime/kuberuntime_container.go:139]: @sourcelliu ^^^^

**Review by mrunalp — APPROVED** [review]

**Pull request merged**